### PR TITLE
Add PowerOff message payload

### PR DESCRIPTION
**Pull request by tuxuser**
Addresses issue #1

### CODE REVIEW

File: src/DarkId.SmartGlass/Messaging/Session/Messages/PowerOffMessage.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+using DarkId.SmartGlass.Common;
+
+namespace DarkId.SmartGlass.Messaging.Session.Messages
+{
+    [SessionMessageType(SessionMessageType.PowerOff)]
+    internal class PowerOffMessage : SessionMessageBase
+    {
+        public string LiveID { get; set; }
```
Comment (joelday) [inline src/DarkId.SmartGlass/Messaging/Session/Messages/PowerOffMessage.cs]: Change to `LiveId`.

**Comment by joelday** [comment]
@tuxuser Awesome. Will need to expose this on `SmartGlassClient` as `PowerOffAsync`, but that's super easy. I assume it doesn't request an ack?

**Comment by tuxuser** [comment]
@joelday Yes, I left the actual implementation for you on purpose.
Indeed, no acknowledge flag is set for this one, console will instantly (sanely tho) power off, no further acknowledge is done.
PS: check msgs

**Comment by joelday** [comment]
@tuxuser Unsure where consumer of the API should acquire the LiveId. What's an example of that value?

**Comment by tuxuser** [comment]
It's part of the x509 certificate of discovery response, "CommonName" field.

**Comment by joelday** [comment]
@tuxuser Nice. I'll finish this one up now.

**Pull request merged**